rest/threads.go

```diff
@@ -22,8 +22,9 @@ type Threads interface {
 	LeaveThread(threadID snowflake.ID, opts ...RequestOpt) error
 	AddThreadMember(threadID snowflake.ID, userID snowflake.ID, opts ...RequestOpt) error
 	RemoveThreadMember(threadID snowflake.ID, userID snowflake.ID, opts ...RequestOpt) error
-	GetThreadMember(threadID snowflake.ID, userID snowflake.ID, opts ...RequestOpt) (threadMember *discord.ThreadMember, err error)
-	GetThreadMembers(threadID snowflake.ID, opts ...RequestOpt) (threadMembers []discord.ThreadMember, err error)
+	GetThreadMember(threadID snowflake.ID, userID snowflake.ID, withMember bool, opts ...RequestOpt) (threadMember *discord.ThreadMember, err error)
+	GetThreadMembers(threadID snowflake.ID, after snowflake.ID, limit int, opts ...RequestOpt) (threadMembers []discord.ThreadMember, err error)
```

Comment:
this endpoint only supports pagination when providing `with_member=true`. So those params do nothing